controller/common/customize/manager.go

```diff
@@ -97,8 +99,8 @@ func (rm *Manager) getRelatedClient(apiVersion, resource string) (*dynamicclient
 	if err != nil {
 		return nil, nil, err
 	}
-
-	informer := rm.relatedInformers.Get(apiVersion, resource)
+	groupVersion, _ := schema.ParseGroupVersion(apiVersion)
```

Comment:
Are we ok to ignore err (i.e. second response)?

Reply:
if something would be wrong with `apiVersion` it would exploded in line 97  (`client, err := rm.dynClient.Resource(apiVersion, resource)`)